wizard-server/src/main/java/org/cobbzilla/wizard/filters/auth/TokenPrincipal.java

```diff
@@ -5,6 +5,6 @@
 public interface TokenPrincipal extends Principal {
 
     String getApiToken ();
-    TokenPrincipal setApiToken (String token);
+    void setApiToken (String token);
```

Comment:
Can we leave this unchanged? It should return `TokenPrincipal`. I prefer setters that `return this`.